test/membrane_http_adaptive_stream/integration_test/sink_bin_integration_test.exs

```diff
@@ -2,146 +2,137 @@ defmodule Membrane.HTTPAdaptiveStream.SinkBinIntegrationTest do
     alias Membrane.HTTPAdaptiveStream.{SinkBin, HLS}
     alias Membrane.HTTPAdaptiveStream.Storages.FileStorage
-    alias Membrane.H264.FFmpeg.Parser, as: H264_Parser
-    alias Membrane.AAC.Parser, as: AAC_Parser
 
     @impl true
-    def handle_init(options) do
+    def handle_init(%{:sources => sources, :output_dir => output_dir}) do
       sink_bin = %SinkBin{
         muxer_segment_duration: 2 |> Membrane.Time.seconds(),
         manifest_module: HLS,
         target_window_duration: 30 |> Membrane.Time.seconds(),
         target_segment_duration: 2 |> Membrane.Time.seconds(),
         persist?: false,
         storage: %FileStorage{
-          directory: options[:output_dir]
+          directory: output_dir
         }
       }
 
-      elements =
-        options[:sources]
-        |> Enum.map(fn {source, encoding, track_name} ->
+      children =
+        sources
+        |> Enum.flat_map(fn {source, encoding, track_name} ->
+          parser =
+            case encoding do
+              :H264 ->
+                %Membrane.H264.FFmpeg.Parser{
+                  framerate: {30, 1},
+                  alignment: :au,
+                  attach_nalus?: true
+                }
+
+              :AAC ->
+                %Membrane.AAC.Parser{
+                  out_encapsulation: :none
+                }
+            end
+
           [
             {{:source, track_name}, source},
-            {{:parser, encoding, track_name}, get_parser_for_encoding(encoding)}
+            {{:parser, encoding, track_name}, parser}
           ]
         end)
+        |> then(&[{:sink_bin, sink_bin} | &1])
 
       links =
-        elements
-        |> Enum.map(fn [
-                         {{:source, name} = source_id, _},
-                         {{:parser, encoding, name} = parser_id, _}
-                       ] ->
-          link(source_id)
-          |> to(parser_id)
-          |> via_in(Pad.ref(:input, name), options: [encoding: encoding, track_name: name])
+        sources
+        |> Enum.map(fn {_source, encoding, track_name} ->
+          link({:source, track_name})
+          |> to({:parser, encoding, track_name})
+          |> via_in(Pad.ref(:input, track_name),
+            options: [encoding: encoding, track_name: track_name]
+          )
           |> to(:sink_bin)
         end)
 
-      children = elements |> List.flatten() |> Enum.into(%{:sink_bin => sink_bin})
-
       {{:ok, spec: %ParentSpec{children: children, links: links}}, %{}}
     end
+  end
 
-    defp get_parser_for_encoding(encoding) do
-      case encoding do
-        :H264 ->
-          %H264_Parser{
-            framerate: {30, 1},
-            alignment: :au,
-            attach_nalus?: true
-          }
-
-        :AAC ->
-          %AAC_Parser{
-            out_encapsulation: :none
-          }
-      end
+  describe "Reference HLS content creation should only take place locally." do
+    test "check if fixture creation is unset" do
+      assert not @create_fixtures
```

Comment:
Appreciate the test, but I think you got the description a little wrong. Locally? What are other options? I would say, if not locally, then remotely and we are not checking that. We are only checking if @create_fixtures is set to false, so that should probably be a description. 

Also, use `refute`:
```suggestion
    test "check if fixture creation is disabled" do
      refute @create_fixtures
```

Reply:
Done